src/components/InputProblem.tsx

```diff
@@ -68,21 +69,45 @@ export const InputProblem = ({
   }, [feedback])
 
   const evaluateInput = (input: string, answer: string): boolean => {
+    input = input.trim()
+    answer = answer.trim()
```

Comment:
Just to avoid potential confusion when reading the code, it might be nicer to do something like:

```javascript
    const trimmedInput = input.trim()
    const trimmedAnswer = answer.trim()
```

and use those variables below.